opendevin/sandbox/docker/exec_box.py

```diff
@@ -122,7 +122,10 @@ def run_command(container, command):
                     self.container.exec_run(
                         f'kill -9 {pid}', workdir=SANDBOX_WORKSPACE_DIR)
                 return -1, f'Command: "{cmd}" timed out'
-        return exit_code, logs.decode('utf-8').strip()
+        logs_out = logs.decode('utf-8')
+        if logs_out.endswith('\n'):
+            logs_out = logs_out[:-1]
```

Comment:
if you are changing `strip()` to this, you'd probably wanna do this in other sandboxes too?

Reply:
TBH I want to redo all the log parsing in a separate PR--I don't think we should be stripping whitespace from log output. This is here because the tests don't run consistently otherwise ☹️